zwave_js_server/model/node.py

```diff
@@ -201,6 +203,11 @@ def neighbors(self) -> List[int]:
         """Return the neighbors."""
         return self.data.get("neighbors", [])
 
+    @property
+    def endpoints(self) -> List[Endpoint]:
+        """Return the endpoints."""
+        return [Endpoint(endpoint) for endpoint in self.data["endpoints"]]
```

Comment:
Will the `endpoints` item always be present in the node data?

Reply:
I believe this means that it does: https://github.com/zwave-js/zwave-js-server/blob/master/src/lib/state.ts#L98

I don't really know TypeScript but I am assuming that worst case scenario, it will be an empty array

Reply:
I agree:
https://zwave-js.github.io/node-zwave-js/#/api/node?id=getallendpoints